ST_defines.h

```diff
@@ -44,12 +44,6 @@
  * \ingroup SPECIES
  */
 #define MAX_SPECIES (SuperGlobals.max_spp_per_grp * SuperGlobals.max_rgroups)
-/**
- * \brief how long the name of a species can be.
- * 
- * \ingroup SPECIES
- */
-#define MAX_SPECIESNAMELEN   4 /* keep around for SOILWAT for now */
```

Comment:
I assume this was deleted because it is defined in SW_defines.h?

Reply:
Yes, this is correct. Sorry -- I only posted the full explanation on the SOILWAT2-side: https://github.com/DrylandEcology/SOILWAT2/commit/b53f0b5e970fb9898863b60fab05bd3bd5ceb680

> 
> - `MAX_SPECIESNAMELEN` is used by module "SW_VegEstab"
> - it was defined by "SW_Define" unless the code was compiled for STEPWAT2, then it was defined by "STEPWAT2/ST_define"
> --> define it all the time by "SW_Define"

Reply:
thanks!